corehq/apps/app_manager/views/download.py

```diff
@@ -240,11 +240,17 @@ def resolve_path(path):
 
     try:
         assert request.app.copy_of
-        obj = CachedObject('{id}::{path}'.format(
-            id=request.app._id,
-            path=full_path,
-        ))
-        if not obj.is_cached():
+        if toggles.NO_CACHE_APP_FILES.enabled(domain):
```

Comment:
How heavily is htis cached? this will cause more hits to redis/couch, but guessing it's still worth it